modules/Microsoft.Network/bastionHosts/.test/parameters.json

```diff
@@ -20,6 +20,18 @@
         "scaleUnits": {
             "value": 4
         },
+        "disableCopyPaste": {
+            "value": true
+        },
+        "enableFileCopy": {
+            "value": false
+        },
+        "enableIpConnect": {
+            "value": false
+        },
+        "enableShareableLink": {
+            "value": false
+        },
```

Comment:
@ChrisSidebotham can we add a min.parameters.json to validate the minimum required parameters for a bastion host? Probably have it as basic sku?

Reply:
We already have a min.parameters.json test in the .tests folder https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.Network/bastionHosts/.test/min.parameters.json 
Is this what you are referring to?

Reply:
Totally missed this one. Sorry for that